module2/homework/maxOfVector/maxOfVector.hpp

```diff
@@ -3,6 +3,11 @@
 #include <limits>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+    int maxVec = vec[0];
+    for (size_t i = 0; i < vec.size(); ++i) {
```

Comment:
You could start from index 1 because you've already used `vec[0]` last line.
```suggestion
    for (size_t i = 1; i < vec.size(); ++i) {
```

Reply:
You are right, I did not notice it. It is changed now